renovate.json

```diff
@@ -4,5 +4,13 @@
   "gomod": {
     "enabled": true
   },
-  "postUpdateOptions": ["gomodTidy"]
+  "postUpdateOptions": ["gomodTidy"],
+  "packageRules": [
+    {
+      "matchPackageNames": ["golang"],
+      "matchFileNames": [".tool-versions", "go.mod", "Dockerfile"],
+      "groupName": "golang",
+      "groupSlug": "golang"
+    }
+  ]
```

Comment:
please add this to shared pipelines once verified working https://github.com/app-sre/shared-pipelines/blob/main/renovate/default.json